etcdserver/etcdserverpb/rpc.proto

```diff
@@ -842,15 +850,19 @@ message Member {
   uint64 ID = 1;
   // name is the human-readable name of the member. If the member is not started, the name will be an empty string.
   string name = 2;
+  // isLearner indicates if the member is raft learner.
+  bool isLearner = 3;
   // peerURLs is the list of URLs the member exposes to the cluster for communication.
-  repeated string peerURLs = 3;
+  repeated string peerURLs = 4;
```

Comment:
I'ts backward incompatible to re-number proto fields since proto only use field numbers, not the field names when serializing/deserializing. isLearner should be assigned a never before used field number.

Reply:
Addressed. Thanks!